modules/webxr/webxr_interface_js.cpp

```diff
@@ -230,6 +231,43 @@ Array WebXRInterfaceJS::get_available_display_refresh_rates() const {
 	return ret;
 }
 
+Array WebXRInterfaceJS::get_supported_environment_blend_modes() {
+	Array blend_modes;
```

Comment:
Maybe add a comment here that users can't switch blend modes, it's predefined at startup, so we return the selected blend mode as the only supported blend mode.

Reply:
Sure, that makes sense! Added in my latest push